PROV/prov-bfo-directmappings.ttl

```diff
@@ -54,36 +54,47 @@ prov:Agent                rdfs:comment "A prov:Agent is a material entity becaus
                                           ] .
 
 
-obo:BFO_0000015           rdfs:comment "A prov:Activity is equivalent to a process because it happens over time, while not being a temporal region itself."@en ;
+obo:BFO_0000015           rdfs:comment "A prov:Activity is equivalent to a process because it occurs over time, while not being a temporal region itself."@en ;
                           owl:equivalentClass prov:Activity .
 
 
-prov:Role                 rdfs:comment "A prov:Role is a bfo:role because its externally determined by the context in which the bearer plays the role."@en ;
-                          rdfs:subClassOf obo:BFO_0000023 . # role TODO: realized in an Activity?
-
-
 prov:Location             rdfs:comment "A prov:location is a subclass of site as the prov:Location term fits under the BFO:site defiition (a three-dimensional immaterial entity that is (partially or wholly) bounded by a material entity or it is a three-dimensional immaterial part) but adds specificity in that it can be an ISO 19112 geographic place or another location in which provenance occurs."@en ;
-                          rdfs:subClassOf obo:BFO_0000029 . # site
+                          owl:equivalentClass obo:BFO_0000029 . # site
```

Comment:
If here we use "owl:equivalentClass", then the above comment need to be changed accordingly.